src/scripts/create-run.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+set -o nounset
+set -o errexit
+set -o xtrace
+
+export RUN_API_TOKEN=${!RUN_API_TOKEN_ENV_NAME}
+cd ${PROJECT_DIR} || exit
+testlio create-run --projectConfig ${PROJECT_CONFIG_PATH} --testConfig ${TEST_CONFIG_PATH}
```

Comment:
@aldomozhirov open question: do you think it could make sense to use the [npx command](https://docs.npmjs.com/cli/v8/commands/npx) directly here instead of having an [install](https://github.com/Testlio/testlio-circleci-orb/pull/1/files#diff-4541ed1902bb35d030557579b31be4c9369bffcf71f691d24e052986a3eb3c89) orb command?

Reply:
From my perspective using install orb command to install CLI probably makes code a bit cleaner. I saw this solution in several other orbs (for example: https://circleci.com/developer/orbs/orb/circleci/github-cli) and in my opinion it makes sense. Imagine, what if we will have more complex interactions with Testlio? I believe, there will be a big amount of code duplication in case of using npx each time.

Reply:
ok, make sense